python/Quickstart.MD

```diff
@@ -5,130 +5,107 @@ This quickstart covers:
+    "deviationType": "UNPLANNED",
+    "materialType": "FINISHED_GOOD",
+    "materialSubtype": "SHIPPED",
+    "materialProblem": "SHORT_SHIPMENT_RECEIVED",
+    "isEscalated": False,
+    "createdByPartner": False,
+    "isSubmittedToPartner": False,
+    "directSupplierImpact": {
+      "businessImpact": "Business Impact",
+      "businessPriority": "HIGH"
+    }
+  }
 }
 ```
+This creates a dictionary object containing the `header` and `payload` elements we discussed earlier.  You can work with the items in this dictionary using standard Python methods.  This allows you to make any desired modifications prior to packaging and submitting your request.  
 
-The `utils.py` file has functions to help build the payload for your request body. The `read_payload_file(filename)` function reads the contents of the file, loads it into a JSON object, and does some minor processing for insertion into the request body.
+As an example, let's change the priority of this incident from `HIGH` to `MEDIUM`.  To change this value, we must reference the `businessPriority` item.  To reach this value, reference the `directSupplierImpact` element of the `payload`.  
 
-In your Python terminal, enter the following:  
+**Type or copy the following into your Python terminal**
 ```
->>> import utils
->>> my_file = 'addIncident.json'
->>> payload_data = utils.read_payload_file(my_file)
+request_body.get('payload').get('directSupplierImpact').update({'businessPriority': 'MEDIUM'})
 ```
-If you already use a JSON payload as part of your other processes, you can the `accept_json_payload(payload_in)` function instead. This function accepts a JSON object, checks for proper format, and does some minor processing for insertion into the request body.
+We use the `.get()` method for each element in our tree that we wish to access.  The `.update()` method allows us to change a value by referencing its key.
+Use care to include the braces `{}` when updating an item.  
 
-In the Python terminal, enter the following:
-```
->>> my_json_data = result_from_some_other_process()
->>> payload_data = utils.accept_json_payload(my_json_data)
+Take a moment to add your `ownerId` and `processNetworkId` values using the method outlined above.  Some sample code is below as a hint:  
 ```
+# establish a couple of variables for convenience later
+ownerId = "YOUR_OWNER_ID"
+network_id = "YOUR_PROCESS_NETWORK_ID"
 
-### Header
+request_body.get('header').update({'ownerId': owner_id})
+request_body.get('header').update({'processNetworkId': network_id})
+```
 
-Now that you have usable payload data, you need to inform APT how to process the payload.  
+### Request Header
+APT-SCWM uses tokens for authentication.  For single interactive requests, you can use a bearer token (a token that usually expires within an hour) or a longer-lived token for automated processes.  The token is a string that contains 'user:password' in Base64 encoding.
```

Comment:
Reword to eliminate ( ) ... suggestion below 

"APT-SCWM uses tokens for authentication. For single interactive requests, use a short-lived bearer token, which lasts for an hour, or a longer-lived token for automated processes." 

You could link back out to auth documentation in repo

Reply:
Updated.  Not linking at this moment.